docs/technical/security.md

```diff
@@ -0,0 +1,201 @@
+# Security Architecture
+
+The architecture for Buerokratt is almost entirely Kubernetes based.  For the MVP we will be using the Azure Kubernetes Service (AKS).
+
+GitHub hold source code for the various components being built and will also host the CI/CD pipelines via GitHub Actions.
+
+![Development Workflow](./DevelopmentWorkflow.editable.png)

```

Comment:
Putting images in images directory would make documents directory a bit easier to read
```suggestion
![Development Workflow](../images/DevelopmentWorkflow.editable.png)

```

Reply:
Yep - good point!